test/Microsoft.AspNet.Mvc.TagHelpers.Test/ScriptTagHelperTest.cs

```diff
@@ -288,8 +326,8 @@ public static TheoryData RunsWhenRequiredAttributesArePresent_Data
             await helper.ProcessAsync(context, output);
 
             // Assert
-            Assert.Null(output.TagName);
```

Comment:
Actually the split for `NoSrc` is same as the split for `output.TagName`. I am adding back these asserts.